syne_tune/optimizer/schedulers/searchers/gp_fifo_searcher.py

```diff
@@ -380,7 +380,7 @@ def _assign_random_searcher(self, random_seed=None):
             if random_seed is None:
                 random_seed = self.random_state.randint(0, 2 ** 32)
             self._random_searcher = RandomSearcher(
-                self.hp_ranges.config_space,
```

Comment:
Because sampling was done from "global" search space for warmstart, initial candidates would be out of bounds for active task.

Reply:
Thanks for catching this!

The best way to fix this is to expose the internal HyperparameterRanges._config_space_for_sampling as a property, and then use that. Do you want me to do this?

Reply:
OK: #169 addresses this